src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.Designer.cs

```diff
@@ -9224,12 +9224,11 @@ internal class Strings {
                 return ResourceManager.GetString("SQL_IntegratedWithUserIDAndPassword", resourceCulture);
             }
         }
-
+        
         /// <summary>
         ///   Looks up a localized string similar to Cannot use &apos;Authentication=Active Directory Interactive&apos; with &apos;User ID&apos;, &apos;UID&apos;, &apos;Password&apos; or &apos;PWD&apos; connection string keywords..
         /// </summary>
-        internal static string SQL_InteractiveWithUserIDAndPassword
-        {
+        internal static string SQL_InteractiveWithUserIDAndPassword {
```

Comment:
Same here. Building the csproj will properly autogenerate this file so that you aren't introducing accidental changes by editing manually.

Reply:
(Amending your first commit is a good idea so that the line history doesn't have extra noise of an accidental change and then a second change to revert the first.)